alws/utils/pulp_utils.py

```diff
@@ -12,10 +15,131 @@
     RpmPackage,
 )
 
+from alws.utils.modularity import IndexWrapper
+
 
 def get_uuid_from_pulp_href(pulp_href: str) -> uuid.UUID:
     return uuid.UUID(pulp_href.split("/")[-2])
 
+def get_modules_yaml_from_repo(repo_name: str):
```

Comment:
2 spaces needed between functions

Reply:
we can move this function somewhere to `utils/`, I assume it can be helpful in the future